intro-OpenStack/ch_LaunchInstances.tex

```diff
@@ -0,0 +1,202 @@
+    from an image or instance snapshot, the image or snapshot will be
+    copied to a volume.  This way, the state of your instance persists
+    after shutdown and reboot.
+  \end{description}
+
+\item[Flavor] tab. Specify the size of the instance to launch.
+
+  \strong{Note:} The flavor is selected based on the size of the image
+  selected for launching an instance. For example, while creating an
+  image, if you have entered the value in the Minimum RAM (MB) field
+  as 2048, then on selecting the image, the default flavor is
+  \textbf{m1.small}.  If a '!' warning sign is displayed next to a
+  resource for one of the flavors, that means that this flavor would
+  exceed the project's quota for that resource, and therefore is not
+  available.
+
+\item[Networks] tab. Add one or more networks to the instance.
+
+\item[Network Ports] tab. Activate the ports that you want to assign to
+  the instance.
+\item[Security Groups] tab. Activate the security groups that you want
+  to assign to the instance.
+
+  Security groups are a kind of cloud firewall that define which
+  incoming network traffic is forwarded to instances.
+
+  The default security group is assigned to the instance
+  automatically.
+\item[Key Pair] tab. Specify a key pair.
+
+  If the image uses a static root password or a static key set
+  (neither is recommended), you do not need to provide a key pair to
+  launch the instance.
+\item[Configuration] tab. Specify a customization script that runs
+  after your instance launches.
+
+\item[Server Groups] tab.
+
+\item[Scheduler Hints] tab.
+
+\item[Metadata] tab. Add Metadata items to your instance.
+\end{description}
+
+\item Click Launch Instance.
+\end{enumerate}
+
+The instance starts on a compute node in the cloud.
+
+\strong{Note:} If you did not provide a key pair, security groups, or
+rules, users can access the instance only from inside the cloud
+through VNC. Even pinging the instance is not possible without an ICMP
+rule configured.
+
+You can also launch an instance from the Images or Volumes category when
+you launch an instance from an image or a volume respectively.
+
+When you launch an instance from an image, OpenStack creates a local
+copy of the image on the compute node where the instance starts.
+
+For details on creating images, see
+\href{https://docs.openstack.org/image-guide/create-images-manually.html}{\emph{Creating
+images manually}} in the \emph{OpenStack Virtual Machine Image Guide}.
+
+\subsection*{Connect to your instance by using SSH}\label{connect-to-your-instance-by-using-ssh}
+
+To use SSH to connect to your instance, use the downloaded keypair file.
+
+\strong{Note:} The user name is \textbf{ubuntu} for the Ubuntu cloud
+images on TryStack.
+
+\begin{enumerate}
+\item Copy the IP address for your instance.
+\item Use the \textbf{ssh} command to make a secure connection to the
+  instance. For example:
+\item \$ ssh -i MyKey.pem ubuntu@10.0.0.2
```

Comment:
we got some questions from the current cloud users, we have to add an specific example for Linux/Mac and Windows to clarify this point.
* For Linux/Mac users:
it is mandatory to use ssh-agent to get access from the login node to the VMs. We do not have to explain this here but just include the section from the HPC introduction to see how to enable the ssh-agen (section 2.4.1 if I'm not mistaken) so for these users:
```
$ ssh -A <vsc_account>@login.hpc.ugent.be
$ ssh -A <os_user>@<public floating ip> or ssh -A root@<public floating ip> (ssh will show the correct user)
```

* Windows users. They must use pageant instead as ssh-agent, fortunately this is also described in our HPC documentation, we do not have to repeat the info, just add a link to HPC introduction for windows (same section 2.1.4 I think). They should connect as usual to the login node using putty and then just from the putty console repeat the same:
$ ssh -A <root or os_user>@<public_ip>

Also we have to explain that the default VM user is included into the sudoers list so they can execute `sudo -s`    or `sudo <command>`   to get admin access.